### DISCUSSION TITLE
buddhism/2024/noble_truths/

### DISCUSSION THREAD

**Discussion by giscus** [shane0/buddhism#89] [General]
# buddhism/2024/noble_truths/

https://shanenull.com/buddhism/2024/noble_truths/

**Comment by shane0** [comment]
# edward consent translation

1 What then is the Holy Truth of Ill? Birth is ill, decay is ill, sickness is ill, death is ill. To be conjoined with what one dislikes means suffering. To be disjoined from what one likes means suffering. Not to get what one wants, also that means suffering. In short, all grasping at (any of) the five skandhas (involves) suffering.

2 What then is the Holy Truth of the Origination of Ill? It is that craving which leads to rebirth, accompanied by delight and greed, seeking its delight now here, now there, i.e. craving for sensuous experience, craving to perpetuate oneself, craving for extinction.

3 What then is the Holy Truth of the Stopping of Ill? It is the complete stopping of that craving, the withdrawal from it, the renouncing of it, throwing it back, liberation from it, non-attachment to it.

4 What then is the Holy Truth of the steps which lead to the stopping of Ill? It is this holy eight-fold Path, which consists of: Right views, right intentions, right speech, right conduct, right livelihood, right effort, right mindfulness, right concentration.”


## others

https://shanenull.com/buddhism/2023/upside_down/

**Reply by shane0** [reply]
https://shanenull.com/buddhism/2022/books/confusion-to-clarity/#four-noble-truths

**Comment by shane0** [comment]
https://youtu.be/_elej83fZ-A?si=C_lLYMVYzNc5f76E

**Comment by shane0** [comment]
hyoshigi
https://youtu.be/i_oBILmSe-U

**Comment by shane0** [comment]
giving is the best communication

https://youtu.be/bL9KEaike8U

**Comment by shane0** [comment]
The Four Noble Truths in Sanskrit are:

1. **Duhkha** (suffering)
2. **Samudaya** (origin of suffering)
3. **Nirodha** (cessation of suffering)
4. **Marga** (path to the cessation of suffering)

**Duhkha**

Duhkha is a Sanskrit word that can be translated as "suffering," "pain," or "unsatisfactoriness." It is the first of the Four Noble Truths, and it refers to the fact that all life is characterized by suffering. Suffering can take many forms, including physical pain, emotional distress, and the impermanence of all things.

**Samudaya**

Samudaya is the second of the Four Noble Truths, and it refers to the origin of suffering. The Buddha taught that suffering arises from craving, or attachment to things that are impermanent and unsatisfactory. When we crave things that we do not have, or when we cling to things that we do have, we create suffering for ourselves.

**Nirodha**

Nirodha is the third of the Four Noble Truths, and it refers to the cessation of suffering. The Buddha taught that suffering can be ceased by the abandonment of craving. When we let go of our attachments, we can free ourselves from suffering.

**Marga**

Marga is the fourth and final of the Four Noble Truths, and it refers to the path to the cessation of suffering. The Buddha taught that the Noble Eightfold Path is the path to the cessation of suffering. The Noble Eightfold Path is a set of eight practices that lead to the development of wisdom, morality, and concentration.

The Four Noble Truths are the foundation of Buddhist teaching. They provide a framework for understanding the nature of suffering and the path to liberation from suffering.
